### DISCUSSION TITLE
OCAPI Promotions call out message

### DISCUSSION THREAD

**Discussion by ajaind1m** [SalesforceCommerceCloud/sfcc-ci#443] [Team Posts]
I have check the OCAPI api and found there is no way to fetch localizable call out message?

https://documentation.b2c.commercecloud.salesforce.com/DOC1/topic/com.demandware.dochelp/OCAPI/19.5/data/Documents/Promotion.html

Do we have any other alternative in OCAPI?

**Comment by mkp4ugithub** [comment]
You may want to try the one from shop API: https://documentation.b2c.commercecloud.salesforce.com/DOC1/topic/com.demandware.dochelp/OCAPI/19.5/shop/Documents/Promotion.html

**Comment by ajaind1m** [comment]
Thanks, I have used this but there is only get method but not put method to set localized message. Any help?
